sdk/storagesync/Microsoft.Azure.Management.StorageSync/CHANGELOG.md

```diff
@@ -1,26 +1,2 @@
-## Microsoft.Azure.Management.StorageSync release notes
+# Release History
 
-### Changes in 2020-03-01
```

Comment:
This is a case where there are no matching headers and we end up throwing out everything. We should figure out about saving this in some sort of empty placeholder.

Reply:
Its management plane so I left it like that.